easybuild/easyconfigs/o/OpenKIM/OpenKIM-1.8.2-foss-2016b.eb

```diff
@@ -0,0 +1,256 @@
+easyblock = 'ConfigureMake'
+name = 'OpenKIM'
+version = '1.8.2'
+
+homepage = 'https://openkim.org/'
+description = """Open Knowledgebase of Interatomic Models.
+
+OpenKIM is an API and a collection of interatomic models (potentials) for
+atomistic simulations.  It is a library that can be used by simulation programs
+to get access to the models in the OpenKIM database.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['kim-api-v%s.tgz' % (version,)]
+
+# Models and drivers are downloaded from the first URL, the API files from the latter.
+source_urls = ['https://openkim.org/download/', 'https://s3.openkim.org/kim-api/']
+
+parallel = 1
+
+modextrapaths = {'KIM_HOME': '.'}
+
+moduleclass = 'chem'
+
+# Sources for all KIM items.  The list will be patched below.
+sources += [
+    #
+    # Model drivers
+    #
+    'MD_853402641673_000.tgz',  # EAM_CubicNaturalSpline
```

Comment:
@schiotz I hope this list was generated somehow? If so, please mention how in comments?